app/src/main/java/com/github/se/project/model/profile/Profile.kt

```diff
@@ -3,7 +3,7 @@ package com.github.se.project.model.profile
 /** Data class representing a user profile. */
 data class Profile(
     val uid: String, // Unique string id for the profile
-    var token: String, // Token for notifications
+    var token: String = "", // Token for notifications
```

Comment:
Well done for setting a default value to avoid to change all the test and depended screens